### PR TITLE
webview menu fix

### PR DESCRIPTION
**Pull request by gopi1410**

### CODE REVIEW

File: app/src/main/java/com/example/ramen/clashroyaleupdates/WebViewActivity.java [hunk 1]
```diff
@@ -1,54 +1,67 @@
...
+            Log.e(TAG, "No URL specified for webview");
             return;
         }
-        link = extras.getString("link");
         Util.openInWebView(mWebPage, link);
     }
 
     @Override
     public boolean onCreateOptionsMenu(Menu menu) {
-        new MenuInflater(this).inflate(R.menu.web, menu);
```
Comment (Rrishik) [inline app/src/main/java/com/example/ramen/clashroyaleupdates/WebViewActivity.java:41]: Why this change??

Comment (gopi1410) [reply app/src/main/java/com/example/ramen/clashroyaleupdates/WebViewActivity.java:41]: This was the one causing the error. This creates a new menu inflater context rather than using the existing one. getMenuInflater() was correctly used in MainActivity, why use new here

Comment (Rrishik) [reply app/src/main/java/com/example/ramen/clashroyaleupdates/WebViewActivity.java:41]: ohk thanks 👍
